openrtb_ext/bidders.go

```diff
@@ -224,6 +224,10 @@ var coreBidderNames []BidderName = []BidderName{
 	BidderZetaGlobalSsp,
 }
 
+func GetAliasBidderNames() map[BidderName]BidderName {
+	return aliasBidderToParent
+}
```

Comment:
From name it implies `GetAliasBidderNames`  will return list of alias bidders. But function actually returns `map[BidderName]BidderName`. How about renaming this function to  `GetAliasToParentMap()`

Reply:
https://github.com/prebid/prebid-server/pull/3055/commits/d5c207f18d210f765ed3301ddcc857da973a8b60